src/test/java/com/clickhouse/kafka/connect/sink/ClickHouseSinkTaskWithSchemaTest.java

```diff
@@ -284,6 +284,33 @@ public void mapTypesTest() {
         assertEquals(sr.size(), countRows(chc, topic));
     }
 
+    @Test
+    public void materializedViewsBug() {
```

Comment:
There might be many bugs. Let's add an issue number instead https://github.com/ClickHouse/clickhouse-kafka-connect/issues/33
```suggestion
    // https://github.com/ClickHouse/clickhouse-kafka-connect/issues/33
    public void materializedViewsIssue33() {
```